src/utils/crt_launch/crt_launch.c

```diff
@@ -359,6 +353,12 @@ int main(int argc, char **argv)
 	MPI_Finalize();
 
 	if (rc == 0) {
+		if (access(g_opt.app_to_exec, F_OK) == -1) {
+			fprintf(stderr, "ERROR: Unable to locate '%s'\n",
+				g_opt.app_to_exec);
+			return -1;
+		}
+
```

Comment:
I'm not sure this will solve the coverity warning, but it's probably helpful anyway.

We should probably be checking the return of execve just below this however.

Reply:
This is for https://scan8.coverity.com/reports.htm#v42092/p12225/fileInstanceId=37283810&defectInstanceId=11265196&mergedDefectId=313043
Basically complains the check and exec of g_opt.app_to_exec are too apart of each other so changes could happen in between, so I moved them closer.

Reply:
This check needs to be reverted to be done where it was originally at; we dont want to do any of MPI stuff if the application is not found. Instead as Ashley mentioned it should just check return from execve and print additional error if it failed. 

Also note that this is a helper utility for launching cart-level tests; it is not used in actual production runs for launching of daos/clients, especially not in situations where security is at stake.

Reply:
I've marked the coverity ticket as ignore due to this being only applicable/used in tests

Reply:
Thanks. Will back out this change.